lib/bpf/bpf_test.go

```diff
@@ -132,29 +124,31 @@ func (s *Suite) TestWatch(c *check.C) {
 		case *apievents.SessionNetwork:
 			pid = ev.BPFMetadata.PID
 		}
-		c.Assert(int(pid), check.Equals, cmd.Process.Pid)
+		require.Equal(t, int(pid), cmd.Process.Pid)
 	}
 }
 
 // TestObfuscate checks if execsnoop can capture Obfuscated commands.
-func (s *Suite) TestObfuscate(c *check.C) {
+func TestObfuscate(t *testing.T) {
```

Comment:
We need to prefix all test names with `TestRoot` as otherwise, CI will run them as a non-root user and all tests will be skipped.
https://github.com/gravitational/teleport/blob/baa2bb2e12b7bec492225aa79784375b1e18bdb4/Makefile#L552-L563